halo/attest/keeper/common_internal_test.go

```diff
@@ -0,0 +1,100 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/omni-network/omni/halo/attest/testutil"
+	"github.com/omni-network/omni/halo/attest/types"
+	"github.com/omni-network/omni/lib/errors"
+
+	storetypes "cosmossdk.io/store/types"
+	"github.com/cosmos/cosmos-sdk/runtime"
+	sdktestutil "github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"go.uber.org/mock/gomock"
+)
+
+type mocks struct {
+	skeeper *testutil.MockStakingKeeper
+	voter   *testutil.MockVoter
+	namer   *testutil.MockChainNamer
+}
+
+func mockDefaultExpectations(_ sdk.Context, m mocks) {}
```

Comment:
This will be refactored in a follow up PR - we need the mocks when we test the public methods. This PR is intentionally smaller and includes only tests of the `Add` method.